mypy/semanal.py

```diff
@@ -2418,22 +2418,25 @@ def visit_await_expr(self, expr: AwaitExpr) -> None:
     # Helpers
     #
 
-    def lookup(self, name: str, ctx: Context) -> SymbolTableNode:
+    def lookup(self, name: str, ctx: Context, include_globals: bool = True,
+               report: bool = True) -> SymbolTableNode:
```

Comment:
`report_errors` will be less ambiguous, so I'll change it.

I intentionally made `include_globals` separate since even though they're only currently used both on or off, someone might later want to use the function in a way where not searching for globals is desired, but reporting an error is. Also, what to search and whether to report an error are totally unrelated features, so this makes the function's behavior clear, precise, and intuitive to its arguments, rather than subtly behaving differently based on a single toggle.

Personally, I'd split `lookup` into different functions for each step, and then have a function where the caller could specify which steps they wanted and whether they wanted an error reported, but that would be an unrelated refactor, so I opted for the least invasive change.